### PR TITLE
Initial recipe

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: .github/workflows/feedstock-merge.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  version_branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "3.8"
```
Comment (cdeepali) [inline .github/workflows/feedstock-merge.yml]:
```suggestion
          python-version: "3.9"
```

File: .github/workflows/feedstock-pr.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  optional_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "3.7"
```
Comment (cdeepali) [inline .github/workflows/feedstock-pr.yml]:
```suggestion
          python-version: "3.9"
```

File: .github/workflows/feedstock-pr.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  required_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "3.7"
```
Comment (cdeepali) [inline .github/workflows/feedstock-pr.yml]:
```suggestion
          python-version: "3.9"
```

**Comment by cdeepali** [comment]
We want to retain the original LICENSE here.

**Review by cdeepali — APPROVED** [review]

**Review by cdeepali — APPROVED** [review]

**Review by npanpaliya — APPROVED** [review]

**Pull request merged**